src/Views/GamePlayViews/EndOfRoundReport/TestClasses/AbbreviatedCard.test.tsx

```diff
@@ -4,12 +4,16 @@ import { act } from 'react-dom/test-utils'
 import AbbreviatedCard from '../AbbreviatedCard'
 
 let container: HTMLElement | null = null
+
+const consoleLog = console.error
 beforeEach(() => {
+  console.error = jest.fn()
   container = document.createElement('div')
   document.body.appendChild(container)
 })
 
 afterEach(() => {
+  console.error = consoleLog
```

Comment:
All I am doing with these changes is make it so that console error messages don't appear in the test result output.